src/pages/api/service/getGameNames.js

```diff
@@ -1,27 +1,30 @@
 import CallIGDB from "./callIGDB.js";
-import GetAgeRating from "./getAgeRatings.js";
 import GetGenres from "./getGenres.js";
 import GetMultiPlayerMode from "./getMultiPlayerMode.js";
 import GetPlatforms from "./getPlatforms.js";
-import GetReleaseDate from "./getReleaseDate.js";
 
 const GetGameName = async () => {
 
     // build up the query string based on these fields
     // extracting ids into easily searchable format for the query string
     // format: (1, 2, 3, 4, ...)
-    const genres = await GetGenres();
-    const platforms = await GetPlatforms();
-    const multiPlayers = await GetMultiPlayerMode();
-    const releaseDate = await GetReleaseDate();
-    const ageRating = await GetAgeRating();
+    // const genres = await GetGenres();
```

Comment:
if the commented code is not needed or duplication lets sanitize the files before merging to staging.

Reply:
Sounds good, I'll do that